### ISSUE TITLE
Auth with Misp Saml

### ISSUE THREAD

**Issue by Prodian0013** [MISP/misp-dashboard#156]
For Misp auth we are using Saml.  I have tried to use the `auth_enabled = True` but it's not redirecting properly.  Running the diagnostic script returns a login error.  Any suggestions for getting Saml to work with misp-dashboard?  Thanks.

**Comment by mokaddem** [comment]
Unfortunately, misp-dashboard does not support MISP custom auth. You'll have to use the `auth_enabled = False` option for it to work. (or propose a PR to make misp-dashboard supports custom auth ;) ).

**Comment by Prodian0013** [comment]
Yeah i did that rewrote the auth to use oauth2 in my fork not sure if anyone else would want that though

**Comment by mokaddem** [comment]
I would say, if you can share the code please do so.
It could be used as a good start if someone wants to build something similar.
Feel free to open a pull request with your changes.
Thanks in advance! :)
